lib/vanagon/platform/windows.rb

```diff
@@ -107,21 +107,22 @@ def generate_msi_package(project)
         target_dir = project.repo ? output_dir(project.repo) : output_dir
         cg_name = "ProductComponentGroup"
         dir_ref = "INSTALLDIR"
+        candle_flags =  "-dPlatform=#{@architecture} -arch #{@architecture} -ext WiXUtilExtension -ext WixUIExtension"
```

Comment:
@melissa and @McdonaldSeanp I don't think the version strings are needed as they are pulled in the ERB file translation in  [product.wxs.erb](https://github.com/puppetlabs/vanagon/blob/master/resources/windows/wix/project.wxs.erb#L9)

@jpogran can you verify and concur?